cosmos/providers/dbt/parser/project.py

```diff
@@ -21,19 +21,58 @@ class DbtModelConfig:
     Represents a single model config.
     """
 
-    tags: set[str] = field(default_factory=set)
+    config_selectors: set[str] = field(default_factory=set)
     upstream_models: set[str] = field(default_factory=set)
 
     def __add__(self, other_config: DbtModelConfig) -> DbtModelConfig:
         """
         Add one config to another. Necessary because configs can come from different places
         """
+
+        # ensures proper order of operations between sql models and properties.yml
+        result = self._config_selector_ooo(
+            prefixes=["materialized", "schema"],
```

Comment:
what do you think about making this a class variable? are we ever going to call `_config_selector_ooo` with a different set of prefixes?

Reply:
I think we'll only add to that prefixes list - take a look at my most recent commit and let me know what you think